about.html

```diff
@@ -0,0 +1,127 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <!-- StyleSheet -->
+    <link rel="stylesheet" href="CSS/style.css">
+    <!-- Script -->
+    <script src="./script.js" defer></script>
+    <title>Webinar: About</title>
+</head>
+<body>
+
+    <!-- Header -->
+    <header class="header">
+        <div class="div_header">
+            <div class="information">
+                <i class="fa-brands fa-twitter"></i>
+                <i class="fa-brands fa-linkedin-in"></i>
+                <a href="#" class="information-link">English</a>
+                <a href="#" class="information-link">My Page</a>
+                <a href="#" class="information-link">Logout</a>
+            </div>
+            <button class="btnMenu" type="button">
+                <span class="material-icons-outlined">menu</span>
+            </button>
+            
+            <nav class="navbar">
+                <div class="left-side">
+                    <a href="index.html"><img src="./Images/wolfPictures.png" alt="logo"></a>
+                </div>
+                <div class="right-side">
+                    <div class="right_div">
+                        <button class="closeMenu" type="button">
+                            <i class="fa-solid fa-xmark"></i>
+                        </button>
+                        <a href="./index.html" class="menu-link">Home</a>
+                        <a href="#" class="menu-link">About</a>
+                        <a href="#" class="menu-link">Program</a>
+                        <a href="#" class="menu-link">Join</a>
+                        <a href="#" class="menu-link">Sponsor</a>
+                        <a href="#" class="menu-link">News</a>
+                        <a href="#" class="menu-link">CC Campain</a>
+                    </div>
+                    
+                </div>
+            </nav>
+            <div class="menu_text">
+                <div class="background_text">
+                    <h2 class="principal_title">About our session</h2>
+                </div>
+                <div class="menu_paragraph">
+                    <p>Contemporary topics such as Privacy, Brand Protection and Managing Digital Investigations, Embracing Leadership That Will Thrive in the Digital Era, Cybersecurity will be top on the agenda. Interactive On-screen presentation with visual aids, Panel discussions will equip professionals with strategies and techniques for rethinking the value of workspaces, workforces, and the work.</p>
+                </div>
+                <div class="date_menu">
+                    <p>Please reach out to us throught the website for the information. Cyber security Webinar 2022! <span>cysecurity2022@gmail.com</span></p>
```

Comment:
Please don't let these paragraphs to stretch so much, and let's just keep the design guidelines:
 Your design:
 
![image](https://user-images.githubusercontent.com/83596135/163605340-8b482789-3ff3-4932-a2a8-d4293b7d7bc7.png)

Provided design:
![image](https://user-images.githubusercontent.com/83596135/163605377-6b1d067b-a7f9-4511-a0bf-0d6a4194c242.png)